cardano-cli/src/Cardano/CLI/Run/Friendly.hs

```diff
@@ -452,6 +456,10 @@ friendlyAuxScripts = \case
   TxAuxScriptsNone -> Null
   TxAuxScripts _ scripts -> String $ textShow scripts
 
+friendlyReferenceInputs :: TxInsReference build era -> Aeson.Value
```

Comment:
json = friendly?

Reply:
Apparently. I was just following the pattern implemented by Yuri. All of it needs to be refactored though.